core/src/main/resources/config.yml

```diff
@@ -5,6 +5,11 @@
 #
 # GitHub: https://github.com/GeyserMC/Geyser
 # Discord: https://discord.geysermc.org/
+# Wiki: https://wiki.geysermc.org/
+#
+# DISCLAIMER: See https://wiki.geysermc.org/geyser/setup/ for the setup guide. Many video tutorials are outdated.
+# In most cases, espescially with server hosting providers, further hosting specific configuration is required.
```

Comment:
Especially

Reply:
oh yea, right
one of the words i find - especially - hard to spell